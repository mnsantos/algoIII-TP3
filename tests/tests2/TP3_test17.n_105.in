105 1788
1 8
1 2
1 7
2 8
1 3
2 7
8 10
1 4
9 10
1 16
5 8
1 9
8 12
1 5
1 10
6 8
6 11
8 14
9 12
1 14
2 5
2 10
7 8
1 18
7 11
9 11
3 9
1 6
1 12
2 15
3 6
3 12
102 103
101 103
100 105
98 105
98 104
98 100
97 103
97 99
97 98
96 98
94 105
94 103
94 100
93 102
93 101
93 99
92 105
92 102
92 101
92 100
92 93
91 102
91 96
91 95
90 103
90 99
90 96
89 105
89 102
89 98
89 96
89 92
88 104
88 92
88 91
88 90
87 105
87 103
87 101
87 100
87 99
87 91
87 89
86 105
86 104
86 95
86 94
86 93
86 88
85 102
85 101
85 100
85 99
85 91
85 86
84 94
84 92
84 90
84 88
84 86
83 95
83 94
83 93
83 88
83 84
82 104
82 95
82 93
82 83
81 105
81 100
81 97
81 95
81 94
81 89
81 88
81 86
81 83
81 82
80 102
80 99
80 96
80 92
80 86
80 81
79 101
79 96
79 93
79 92
79 90
79 85
79 84
78 100
78 97
78 92
78 87
78 86
78 82
78 80
78 79
77 104
77 103
77 102
77 98
77 93
77 91
77 88
76 105
76 103
76 102
76 100
76 95
76 93
76 90
76 84
76 77
75 100
75 98
75 94
75 84
75 80
75 78
74 105
74 102
74 101
74 100
74 95
74 92
74 88
74 85
74 84
74 83
74 82
74 80
74 79
73 105
73 103
73 99
73 95
73 93
73 92
73 81
73 78
73 77
73 76
73 75
72 104
72 92
72 91
72 90
72 88
72 83
72 82
72 80
72 77
72 76
72 75
72 74
71 105
71 99
71 94
71 92
71 89
71 87
71 84
71 72
70 104
70 102
70 101
70 98
70 94
70 88
70 86
70 85
70 81
70 79
70 77
70 73
69 100
69 97
69 96
69 94
69 91
69 89
69 81
69 80
69 78
69 77
69 75
68 105
68 103
68 102
68 99
68 94
68 93
68 92
68 84
68 83
68 82
68 81
68 79
68 75
68 69
67 105
67 104
67 100
67 97
67 95
67 93
67 92
67 91
67 89
67 88
67 86
67 79
67 75
67 72
67 71
67 70
67 68
66 103
66 99
66 97
66 96
66 95
66 93
66 90
66 89
66 84
66 77
66 72
66 71
65 103
65 102
65 87
65 85
65 81
65 74
64 104
64 102
64 101
64 99
64 92
64 86
64 82
64 80
64 79
64 78
64 74
64 71
64 68
24 67
24 66
24 62
24 59
24 56
24 51
24 48
24 44
24 34
24 32
23 102
23 98
23 92
23 89
23 87
23 86
23 83
23 64
23 58
23 55
23 53
23 52
23 49
23 42
23 40
23 39
23 34
23 31
23 29
23 28
23 27
22 104
22 99
22 98
22 97
22 94
22 89
22 87
22 85
22 82
22 74
22 71
22 70
22 68
22 65
22 62
22 58
22 57
22 46
22 42
22 36
22 33
22 32
22 30
22 28
22 24
21 102
21 100
21 98
21 97
21 96
21 92
21 91
21 90
21 88
21 86
21 81
21 80
21 70
21 69
21 67
21 65
21 62
21 55
21 53
21 52
21 47
21 43
21 42
21 29
21 27
21 24
20 100
20 97
20 89
20 86
20 78
20 75
20 72
20 68
20 65
20 62
20 56
20 54
20 53
20 52
20 44
20 42
20 39
20 34
20 28
20 27
20 24
20 21
19 100
19 96
19 91
19 87
19 86
19 82
19 81
19 80
19 78
19 76
19 71
19 69
19 65
19 64
19 63
19 58
19 57
19 56
19 51
19 47
19 46
19 32
19 29
19 25
19 23
19 22
19 21
18 104
18 102
18 98
18 94
18 92
18 89
18 88
18 86
18 75
18 74
18 73
18 67
18 64
18 63
18 61
18 57
18 55
18 51
18 50
18 46
18 45
18 44
18 41
18 40
18 39
18 28
18 22
18 21
18 20
17 105
17 102
17 101
17 97
17 96
17 94
17 92
17 91
17 85
17 84
17 83
17 79
17 77
17 76
17 66
17 64
17 60
17 59
17 58
17 52
17 51
17 50
17 49
17 46
17 36
17 33
17 31
17 27
17 24
17 23
17 22
17 21
16 104
16 103
16 92
16 91
16 90
16 87
16 85
16 84
16 83
16 81
16 78
16 74
16 73
16 72
16 71
16 67
16 65
16 64
16 61
16 58
16 57
16 51
16 49
16 41
16 39
16 31
16 30
16 28
16 26
16 25
16 22
16 18
15 105
15 102
15 100
15 99
15 94
15 93
15 92
15 89
15 88
15 86
15 78
15 75
15 72
15 69
15 56
15 55
15 53
15 51
15 48
15 47
15 46
15 42
15 40
15 30
15 29
15 28
15 23
15 19
14 104
14 101
14 98
14 97
14 93
14 92
14 85
14 84
14 83
14 76
14 72
14 67
14 65
14 64
14 61
14 60
14 51
14 50
14 48
14 42
14 39
14 37
14 33
14 31
14 30
14 26
14 23
14 20
13 92
13 88
13 86
13 85
13 80
13 77
13 71
13 68
13 62
13 59
13 58
13 53
13 51
13 47
13 43
13 39
13 35
13 34
13 32
13 27
13 25
13 22
12 105
12 101
12 99
12 95
12 92
12 90
12 83
12 78
12 77
12 74
12 72
12 71
12 65
12 63
12 60
12 57
12 54
12 50
12 48
12 44
12 39
12 37
12 34
12 29
12 25
12 16
12 15
11 104
11 92
11 90
11 82
11 81
11 80
11 78
11 77
11 76
11 70
11 66
11 65
11 63
11 53
11 51
11 47
11 45
11 43
11 42
11 41
11 38
11 34
11 32
11 31
11 26
11 25
11 22
11 13
10 103
10 102
10 98
10 95
10 94
10 93
10 92
10 85
10 76
10 71
10 70
10 65
10 64
10 63
10 60
10 54
10 52
10 51
10 49
10 48
10 47
10 46
10 45
10 41
10 40
10 39
4 66
4 65
4 64
4 60
4 58
4 54
4 50
4 48
4 47
4 45
4 35
4 32
4 29
4 28
4 27
4 21
4 20
4 18
3 102
3 99
3 96
3 93
3 92
3 88
3 86
3 84
3 83
3 81
3 72
3 67
3 65
3 62
3 60
3 59
3 48
3 47
3 44
3 43
3 40
3 36
3 34
3 33
3 29
3 26
3 21
3 18
2 103
2 99
2 96
2 93
2 87
2 86
2 83
2 80
2 77
2 73
2 72
2 65
2 64
2 59
2 53
2 44
2 41
2 37
2 36
2 35
2 34
2 30
2 28
2 26
2 24
2 22
1 105
1 104
1 103
1 102
1 101
1 100
1 99
1 50
1 49
1 48
1 47
1 46
1 45
1 44
1 43
1 42
1 41
1 40
1 39
1 38
1 37
1 36
1 35
1 34
1 33
1 32
1 31
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
1 51
1 52
1 53
1 54
1 55
1 56
1 57
1 58
1 59
1 60
1 61
1 62
1 63
1 64
1 65
1 66
1 67
1 68
1 69
1 70
1 71
1 72
1 73
1 74
1 75
1 76
1 77
1 78
1 79
1 80
1 81
1 82
1 83
1 84
1 85
1 86
1 87
1 88
1 89
1 90
1 91
1 92
1 93
1 94
1 95
1 96
1 97
1 98
4 70
4 72
4 73
4 75
4 78
4 83
4 84
4 91
4 94
4 95
4 97
4 98
5 17
5 23
5 35
5 40
5 43
5 44
5 47
5 50
5 53
5 56
5 57
5 60
5 62
5 68
5 69
5 73
5 74
5 78
5 79
5 85
5 89
5 90
5 93
5 94
5 99
5 101
5 102
5 104
6 16
6 19
6 21
6 22
6 25
6 26
6 29
6 31
6 32
6 44
6 48
6 54
6 55
6 71
6 72
6 79
6 80
6 81
6 85
6 86
6 92
6 93
6 94
6 97
6 102
7 19
7 24
7 25
7 26
7 28
7 31
7 35
7 37
7 40
7 41
7 42
7 46
7 47
7 48
7 56
7 59
7 60
7 65
7 69
7 72
7 74
7 79
7 82
7 87
7 88
7 89
7 92
7 94
7 96
7 98
7 99
7 101
7 104
7 105
8 19
8 21
8 25
8 28
8 29
8 32
8 36
8 38
8 39
8 44
8 46
8 48
8 51
8 56
8 57
8 60
8 63
8 66
8 67
8 69
8 71
8 72
8 74
8 78
8 90
8 92
8 94
8 99
8 101
8 105
9 19
9 20
9 21
9 24
9 25
9 26
9 29
9 31
9 40
9 42
9 44
9 47
9 50
9 54
9 55
9 59
9 61
9 63
9 66
9 67
9 68
9 69
9 75
9 77
9 78
9 79
9 84
9 85
9 88
9 91
9 92
9 95
9 98
9 99
9 101
9 102
9 103
10 16
10 20
10 23
10 24
10 25
10 27
10 31
10 32
10 36
24 71
24 74
24 80
24 81
24 84
24 87
24 94
24 95
24 96
24 97
24 98
24 102
24 103
24 105
25 26
25 29
25 30
25 31
25 39
25 42
25 45
25 48
25 52
25 54
25 63
25 72
25 79
25 80
25 81
25 82
25 90
25 91
25 93
25 97
25 98
25 101
26 29
26 34
26 37
26 38
26 44
26 45
26 54
26 56
26 61
26 63
26 69
26 70
26 71
26 72
26 75
26 76
26 79
26 80
26 85
26 94
26 95
26 98
26 103
27 28
27 30
27 31
27 33
27 38
27 40
27 45
27 46
27 49
27 53
27 61
27 62
27 67
27 69
27 74
27 76
27 84
27 85
27 89
27 90
27 94
27 97
27 102
28 38
28 42
28 43
28 45
28 46
28 49
28 55
28 56
28 63
28 64
28 67
28 69
28 71
28 73
28 76
28 87
28 88
28 90
28 93
28 97
28 98
28 99
28 102
28 105
29 30
29 31
29 35
29 40
29 43
29 45
29 50
29 51
29 54
29 55
29 56
29 57
29 58
29 61
29 62
29 64
29 65
29 73
29 74
29 75
29 77
29 81
29 82
29 85
29 87
29 88
29 93
29 97
29 98
29 99
29 101
29 102
30 31
30 32
30 35
30 41
30 48
30 52
30 54
30 56
30 57
30 61
30 62
30 65
30 71
30 73
30 74
30 75
30 83
30 87
30 88
30 89
30 94
30 96
30 98
30 99
30 100
30 103
30 104
31 34
31 35
31 37
31 47
31 51
31 60
31 62
31 65
31 73
31 76
31 78
31 79
31 86
31 88
31 89
31 90
31 96
31 104
32 33
32 34
32 36
32 38
32 39
32 41
32 46
32 49
32 53
32 54
32 62
32 63
32 64
32 65
32 68
32 73
32 76
32 78
32 79
32 83
32 92
32 94
32 96
32 98
32 99
32 104
33 34
33 35
33 37
33 40
33 43
33 45
33 49
33 54
33 61
33 65
33 72
33 78
33 87
33 88
33 90
33 93
33 94
33 96
33 102
33 104
33 105
34 35
34 38
34 39
34 41
34 42
34 45
34 49
34 50
34 52
34 55
34 63
34 64
34 66
34 69
34 74
34 86
34 87
34 88
34 92
34 94
34 95
34 96
34 97
34 99
34 101
34 102
35 36
35 38
35 40
35 50
35 54
35 56
35 58
35 64
35 66
35 73
35 87
35 88
35 95
35 103
36 37
36 45
36 46
36 48
36 49
36 53
36 57
36 59
36 66
36 67
36 69
36 72
36 80
36 81
36 83
36 84
36 91
36 92
36 95
36 101
36 104
37 38
37 50
37 51
37 57
37 59
37 64
37 77
37 78
37 82
37 83
37 84
37 86
37 89
37 93
37 94
37 96
37 97
37 99
37 101
37 102
37 103
37 104
38 42
38 46
38 47
38 49
38 52
38 55
38 57
38 58
38 59
38 60
38 62
38 63
38 65
38 66
38 69
38 70
38 73
38 74
38 75
38 78
38 81
38 86
38 88
38 93
38 96
38 101
38 103
38 104
39 41
39 42
39 46
39 49
39 55
39 56
39 58
39 59
39 60
39 63
39 64
39 65
39 67
39 68
39 72
39 74
39 76
39 77
39 80
39 82
39 88
39 89
39 90
39 92
39 104
40 41
40 42
40 47
40 51
40 53
40 54
40 57
40 60
40 65
40 66
40 69
40 70
40 72
40 74
40 75
40 77
40 78
40 81
40 82
40 85
40 90
40 94
40 95
40 97
40 101
40 102
40 105
41 43
41 49
41 51
41 52
41 53
41 60
41 62
41 63
41 68
41 72
41 76
41 80
41 81
41 85
41 88
41 89
41 90
41 97
41 105
42 45
42 54
42 55
42 56
42 59
42 60
42 64
42 67
42 72
42 79
42 89
42 91
42 97
43 45
43 46
43 48
43 50
43 52
43 62
43 64
43 66
43 67
43 68
43 73
43 79
43 90
43 91
43 94
43 96
43 102
43 103
43 104
43 105
44 52
44 57
44 59
44 61
44 63
44 64
44 67
44 68
44 69
44 72
44 73
44 76
44 78
44 79
44 82
44 87
44 95
44 96
44 97
44 98
44 101
44 103
45 49
45 56
45 65
45 66
45 67
45 68
45 69
45 71
45 73
45 75
45 76
45 80
45 86
45 96
45 103
46 48
46 49
46 50
46 54
46 55
46 57
46 60
46 63
46 67
46 78
46 80
46 81
46 83
46 87
46 88
46 89
46 94
46 95
46 97
46 98
46 100
46 103
47 49
47 51
47 52
47 54
47 55
47 59
47 64
47 65
47 66
47 74
47 76
47 79
47 82
47 83
47 87
47 90
47 92
47 95
47 96
47 97
47 98
47 99
47 101
47 102
47 104
47 105
48 53
48 54
48 55
48 57
48 67
48 69
48 70
48 74
48 80
48 81
48 86
48 87
48 88
48 89
48 91
48 93
48 94
48 97
48 98
48 102
48 105
49 50
49 51
49 57
49 58
49 64
49 66
49 67
49 68
49 73
49 75
49 80
49 88
49 96
49 98
49 103
50 52
50 60
50 64
50 67
50 79
50 80
50 82
50 87
50 90
50 91
50 92
50 94
50 95
50 98
50 99
50 103
50 104
50 105
51 64
51 68
51 71
51 75
51 78
51 80
51 81
51 88
51 91
51 95
51 102
51 104
51 105
52 55
52 57
52 61
52 62
52 71
52 75
52 76
52 80
52 86
52 91
52 92
52 93
52 98
52 99
52 105
53 54
53 57
53 67
53 75
53 77
53 80
53 86
53 87
53 91
53 92
53 95
53 97
53 98
53 99
53 100
53 105
54 60
54 67
54 70
54 74
54 75
54 76
54 78
54 82
54 90
54 91
54 92
54 95
54 97
54 100
54 102
54 105
55 59
55 61
55 64
55 69
55 70
55 71
55 72
55 74
55 75
55 76
55 78
55 79
55 90
55 96
55 97
55 101
55 102
55 103
56 65
56 66
56 67
56 68
56 73
56 78
56 84
56 87
56 93
56 94
56 98
57 61
57 65
57 67
57 69
57 71
57 72
57 73
57 85
57 86
57 88
57 89
57 94
57 103
58 61
58 62
58 65
58 66
58 68
58 69
58 70
58 71
58 73
58 75
58 83
58 89
58 91
58 92
58 102
58 104
59 61
59 62
59 67
59 85
59 87
59 89
59 91
59 93
59 95
59 99
59 100
59 102
59 104
60 61
60 70
60 74
60 80
60 81
60 84
60 87
60 89
60 92
60 94
60 96
60 100
60 101
61 62
61 63
61 64
61 65
61 71
61 74
61 77
61 79
61 80
61 82
61 88
61 101
62 64
62 68
62 74
62 78
62 83
62 89
62 90
62 91
62 97
62 101
62 102
62 104
63 66
63 67
63 69
63 70
63 75
63 77
63 81
63 88
63 92
63 93
63 94
63 95
63 96
63 97
63 102
64 67
2 14
4 16
3 10
1 17
5 15
10 11
4 9
1 11
1 15
2 6
3 11
6 12
9 13
5 6
3 13
11 12
3 14
1 13
4 5
5 16
0
