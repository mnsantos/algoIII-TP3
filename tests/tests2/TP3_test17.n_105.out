314 7 1 3 9 10 47 92 102
